cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonLanguage.kt

```diff
@@ -228,6 +229,12 @@ class PythonLanguage :
         return super.tryCast(type, targetType, hint, targetHint)
     }
 
+    override fun isBuiltinsFile(file: File): Boolean {
+        return fileExtensions.any {
+            file.path == IDENTIFIER_INIT + language.namespaceDelimiter + it
```

Comment:
This confuses me. Why is `__init__.py` a built-in ? Also we should not use the namespace delimiter as the file extension "dot"